### PR TITLE
Created HomePolicy class and made some changes to Home class

### PR DESCRIPTION
**Pull request by julia-parewick**
included getters and special method in home class

### CODE REVIEW

File: src/Home.java [hunk 1]
```diff
@@ -24,7 +24,7 @@ private enum Location {
     /**
      * Age of dwelling
      */
-    private LocalDate yearBuilt;
+    private int yearBuilt;
```
Comment (MasonSeward1) [inline src/Home.java:27]: @julia-parewick Why was this changed to an int?

**Review by MasonSeward1 — CHANGES_REQUESTED** [review]

File: src/Home.java (hunk 1 above)
Comment (julia-parewick) [reply src/Home.java:27]: I think it would make input easier. Josh has it as localDate but i didn't think it was overly necessary

**Review by MasonSeward1 — APPROVED** [review]
